src/components/side-panel/common/dataset-tag.js

```diff
@@ -41,11 +44,36 @@ const DatasetTagWrapper = styled.div`
   }
 `;
 
+const DatasetColorPicker = styled.div`
+  display: flex;
+`;
+
+const UpdateDatasetColor = ({id, updateDatasetColor = nop, children}) => (
+  <DatasetColorPicker
+    className="dataset-action update-color"
+    data-tip
+    data-for={`update-color-${id}`}
+    onClick={e => {
+      e.stopPropagation();
+      updateDatasetColor(id);
+    }}
+  >
+    {children}
+    <Tooltip id={`update-color-${id}`} effect="solid">
```

Comment:
make rendering if `Tooltip` optional if `updateDatasetColor` is not provided. So that when the parent component doesn't provide dateDatasetColor prop, the color square will not be clickable

kepler renders `DatasetTag` in other places, but doesn't allow user to edit the color
<img width="340" alt="Screen Shot 2021-09-12 at 6 32 01 PM" src="https://user-images.githubusercontent.com/3605556/133011369-05593948-6642-4068-acc3-c6a397fbaf23.png">